docs/docs/references/customization/config-file.md

```diff
@@ -163,6 +163,23 @@ db:
   java-repository: ghcr.io/aquasecurity/trivy-java-db
 ```
 
+## Registry Options
```

Comment:
1) I think we can add information about these flags to [Docker Private Registries](https://aquasecurity.github.io/trivy/v0.38/docs/advanced/private-registries/)
2) we can also add info about support of multiple credentials.

Reply:
Updated https://github.com/aquasecurity/trivy/pull/3906/commits/3cdc7c2544e7f3d55da1f0d7b13d8716dcd5edb9